docs/CHANGELOG.asciidoc

```diff
@@ -34,6 +35,46 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.6.2]]
+== {kib} 7.6.2
+
+See <<breaking-changes-7.6,breaking changes in 7.6>>.
+
+[float]
+[[bug-7.6.2]]
+=== Bug fixes
+Discover::
+* Fixes infinite loop for view single document on discover {pull}60236[#60236]
+Machine Learning::
+* Shows view series link in anomalies table for machine_learning_user role {pull}59549[#59549]
+* Fixes bucket span estimators loading of max_buckets setting {pull}59639[#59639]
+Management::
+* Handles improperly defined Watcher Logging Action text parameter. {pull}60169[#60169]
```

Comment:
Remove ending period

RN description:

Improperly defined Watcher Logging Action text parameters will now be handled elegantly by Kibana instead of causing Kibana to crash.